src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -167,6 +169,15 @@ const ContentListTable = () => {
   const introspectRepoForUuid = (uuid: string): Promise<void> =>
     introspectRepository({ uuid: uuid, reset_count: true } as IntrospectRepositoryRequestItem);
 
+  const { mutateAsync: triggerSnapshotMutation } = useTriggerSnapshot(queryClient);
+  triggerSnapshotMutation;
+  const { mutateAsync: introspectSnapshotMutation } = useTriggerSnapshot(queryClient);
+  introspectSnapshotMutation;
```

Comment:
delete these two lines, it is basically a copy of the above code with a name that combines snapshot/introspect. 

In this case, "introspectSnapshotMutation" equals "triggerSnapshotMutation" in use. 
and again, you have pasted the function instance "introspectSnapshotMutation" below the query that made the function. 
This wouldn't result in anything, maybe it would error, but that's all.